src/core/include/openvino/core/preprocess/resize_algorithm.hpp

```diff
@@ -7,7 +7,14 @@
 namespace ov {
 namespace preprocess {
 
-enum class ResizeAlgorithm { RESIZE_LINEAR, RESIZE_CUBIC, RESIZE_NEAREST };
+/// \brief An enum containing all supported resize(interpolation) algorithms available in preprocessing
+enum class ResizeAlgorithm {
+    RESIZE_LINEAR,           //!< Linear interpolation matching the TensorFlow behavior
+    RESIZE_CUBIC,            //!< Cubic interpolation
+    RESIZE_NEAREST,          //!< Nearest interpolation
+    RESIZE_BILINEAR_PILLOW,  //!< Bilinear interpolation matching the Pillow behavior
+    RESIZE_BICUBIC_PILLOW    //!< Bicubic interpolation matching the Pillow behavior
```

Comment:
is any plugin support this type of interpolation?

Reply:
CPU will when this PR is merged https://github.com/openvinotoolkit/openvino/pull/16698